Global/Eclipse.gitignore

```diff
@@ -49,3 +49,6 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+
+# Scala IDE specific (Scala & Java development for Eclipse)
```

Comment:
@sherzodv yes, please move any IDE-specific rules to the relevant configuration files under `Global/`. This prevents duplication and makes maintenance easier.

This is also mentioned in the [`CONTRIBUTING.md`](https://github.com/github/gitignore/blob/master/CONTRIBUTING.md) document:

> **Please only modify one template per pull request.** This helps keep pull requests and feedback focused on a specific project or technology.